docs/content/rest-api.md

```diff
@@ -1513,6 +1513,330 @@ The following table summarizes the behavior for partial evaluation results.
 
 > The partially evaluated queries are represented as strings in the table above. The actual API response contains the JSON AST representation.
 
+
+## Health API
+
+The `/health` API endpoint executes a simple built-in policy query to verify
+that the server is operational. Optionally it can account for bundle activation as well
+(useful for "ready" checks at startup).
+
+#### Query Parameters
```

Comment:
I'm not sure that this paragraph is rendered as desired:
![image](https://user-images.githubusercontent.com/870638/145179763-76a33b25-724a-43ba-a2fc-bec763e3ba2d.png)

I think an extra newline between each parameter would do the trick, if we want to avoid another itemization.